src/js/components/Index.js

```diff
@@ -98,38 +111,83 @@ export default class Index extends Component {
     }
     const ViewComponent = VIEW_COMPONENT[view];
 
+    let filterControl;
+    let { filtersInline } = this.props;
+    const { inlineFilterOpen } = this.state;
+
+    // mobile view doesn't get inline filter
+    filtersInline = filtersInline && window.innerWidth >= Responsive.smallSize();
```

Comment:
Responsive needs to be moved to `componentDidMount`, this code would not work in a server side rendered setting, mainly because we are doing DOM manipulation inside Responsive util.